src/engine/server/server.cpp

```diff
@@ -714,6 +714,7 @@ void CServer::SendMap(int ClientID)
 	Msg.AddInt(m_CurrentMapSize);
 	Msg.AddInt(m_MapChunksPerRequest);
 	Msg.AddInt(MAP_CHUNK_SIZE);
+	Msg.AddRaw(&m_CurrentMapSha256, sizeof(m_CurrentMapSha256));
```

Comment:
You could check if the client version is older than 0.7.3 and then skip sending the sha256.